README.md

```diff
@@ -47,30 +47,37 @@ via lektor (see [Deployment](#deployment), below).
 
 ### Pre-Requisites
 
-Make sure you have [pipenv][pipenvdocs], [Node.js][nodejswebsite] and [npm][npmdocs] installed. To install these, execute the following commands:
+Make sure you have:
+
+* [pipenv][pipenvdocs]
+
+* [Node.js][nodejswebsite] **v12+** and [npm][npmdocs] installed. 
+
+To install these, execute the following commands:
 
 > For  macOS:
+
 1. `brew install pipenv`
 2. `brew install node`
 
 > For GNU/Linux:
+
 1. `curl https://raw.githubusercontent.com/kennethreitz/pipenv/master/get-pipenv.py | python`
 2. `sudo apt update`
-3. `sudo apt install nodejs`
-4. `sudo apt install npm `
+3. `curl -sL https://deb.nodesource.com/setup_lts.x | sudo -E bash -`
+4. `sudo apt-get install -y nodejs`
+5. `sudo apt install npm `
 
 [pipenvdocs]:https://pipenv.readthedocs.io/en/latest/
 [nodejswebsite]:https://nodejs.org/en/
 [npmdocs]:https://docs.npmjs.com/ 
 
-
 ### Installing Project Requirements
 
 1. Clone this repository.
-2. Open your command line interface and `cd` to the repository root directory.
+2. Open your command line interface and `cd` to the repository's root directory.
```

Comment:
Not to be pedantic here but `'s` isn't needed because 'repository root' is a perfectly valid grammatical construct. Two nouns together can form a noun phrase where the first noun acts as an adjective to modify the second.